resources/extruders/winbo_superhelper105_extruder.def.json

```diff
@@ -0,0 +1,19 @@
+{
+    "id": "winbo_superhelper105_extruder",
+    "version": 2,
+    "name": "Extruder 1",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "winbo_superhelper105",
+        "position": "0"
+    },
+
+    "overrides": {
+        "extruder_nr": {
+            "default_value": 0,
+            "maximum_value": "2"
+        },
+        "machine_nozzle_offset_x": { "default_value": 0.0 },
+        "machine_nozzle_offset_y": { "default_value": 0.0 }
+    }
```

Comment:
As I commented in PR #4117 , you need to specify here the `material_diameter` and `machine_nozzle_size` if they are different than the defaults (2.85mm and 0.4mm, respectively)